src/_common/Tab.js

```diff
@@ -21,13 +21,13 @@ export default class Tab extends Component {
     };
 
     render() {
-        const { imgSrc, selected, showIcon, showText, text, onMouseDown, onClose, closable } = this.props;
+        const { imgSrc, selected, showIcon, showText, text, onClick, onClose, closable } = this.props;
 
         return (
             <div
                 role="tab"
                 aria-selected={selected}
-                onMouseDown={onMouseDown}
+                onClick={onClick}
```

Comment:
@qingweibinary sounds like one of the events is propagating or isn't it ?  Perhaps you should stopPropagation and preventDefault on events . 😉